docs/topics/Data-Verification.md

```diff
@@ -0,0 +1,62 @@
+# Data Verification
+
+Submitted tournaments run through a series of manual and automated checks before they may be processed for rating calculations. The [](DataWorkerService.md) and [](Automated-Checks.md) pages describe most of the automatic flow, while this page details the manual flow for verifying tournament data and provides more transparency on various conditions. 
+
+## Verifying or rejecting data
+
+Once data finishes running through automated checks, each game score, game, match, and tournament will be marked as either `PreVerified` or `PreRejected`. Typically, `PreVerified` data will end up `Verified` and `PreRejected` data will end up `Rejected` after manual review, and the steps below explain how the o!TR team interprets or modifies these statuses.
+
+<procedure>
+<step>
+First, general information about the tournament (rank limit, lobby size, any gimmick) is cross-checked against the tournament's wiki page or forum post. If the tournament does not meet the <a href="Tournament-Approval.md#acceptance-criteria">acceptance criteria</a>, it will be rejected, meaning that none of the match data will factor into ratings.
+<tip>
+There is currently no strict numerical cutoff for allowed forfeits or minimum match count per tournament. If there is a particularly small bracket or high percentage of FFs, an admin note will detail the reason for verification or rejection (which will reflect whether the tournament was played in a reasonably competitive environment).
+</tip>
+</step>
+<step>
+Next, matches that have been <code>PreRejected</code> or otherwise flagged are considered. If the data is valid but there is a mismatched acronym or other improper formatting (e.g. improper conversion of a head-to-head match to Team VS), those issues may be overridden. If there are an usually small number of games (4 or fewer), verification depends on whether the match was forfeited early. 
+
+<tip>
+There may be some ambiguity with partial matches, since they may come from the loser conceding early (in which case the match should be generally verified) or complications due to technical issues or missing players (in which case it may be generally rejected). When in doubt, the original sources of truth -- the tournament bracket or main sheet -- are consulted.
```

Comment:
```suggestion
There may be some ambiguity with partial matches, since they may come from the loser conceding early, or as a result of technical issues or missing players. When in doubt, the original sources of truth -- the tournament bracket or main sheet -- are consulted.
```